tests/e2e/report-submission.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test, expect, chromium } from "@playwright/test"
+import login from "./utils/login"
+
+test.describe("Authenticated Dispatch App", () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page)
+  })
+  test("Should allow me to report an incident", async ({ page }) => {
+    /* The ability to report an incident is one of the most critical
+      user stories in the Dispatch application. */
+
+    const project = "Test"
+    const type = "Customer Data"
+    const priority = "Low"
+
+    await page.getByRole("link", { name: "Report Incident" }).click()
```

Comment:
I wonder if there might be a reason to visit the URL directly as opposed to clicking the `Report Incident` button. I suspect the direct URL visit may be more reliable and this test might be better isolated if we avoid any side effects of this button click.